x/trigger/types/keys.go

```diff
@@ -0,0 +1,150 @@
+package types
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+	fmt "fmt"
+	"strings"
+)
+
+const (
+	// ModuleName defines the module name
+	ModuleName = "trigger"
+
+	// StoreKey defines the primary module store key
+	StoreKey = ModuleName
+
+	// RouterKey is the message route for slashing
+	RouterKey = ModuleName
+
+	// QuerierRoute defines the module's query routing key
+	QuerierRoute = ModuleName
+
+	TriggerIDLength  = 8
+	QueueIndexLength = 8
+	GasLimitLength   = 8
+)
+
+var (
+	TriggerKeyPrefix       = []byte{0x01}
+	EventListenerKeyPrefix = []byte{0x02}
+	QueueKeyPrefix         = []byte{0x03}
+	GasLimitKeyPrefix      = []byte{0x04}
+	NextTriggerIDKey       = []byte{0x05}
+	QueueStartIndexKey     = []byte{0x06}
+	QueueLengthKey         = []byte{0x07}
```

Comment:
It's immensely useful to know the key formats without having to wade through the key generation functions. Usually we put them as comments in this area.

They should also be included in the state spec documentation. Basically, "here's the key format and here's the value format." The "value format" is often just a proto message, so that's easy, but it's really nice to have the key formats written down too.